### DISCUSSION TITLE
Problem while run the server

### DISCUSSION THREAD

**Discussion by ababeel-ceo** [chartbrew/chartbrew#208] [Q&A]
Hello Guys,

I've followed the steps to run the project but when i run server it through these errors but i've done the database creation and .env files perfectly. But don't know why this error came. pls help me out guys

Running server on port 4019
node:internal/process/promises:288
            triggerUncaughtException(err, true /* fromPromise */);
            ^

Error
    at Query.run (D:\Test\chartbrew\server\node_modules\sequelize\lib\dialects\postgres\query.js:50:25)
    at D:\Test\chartbrew\server\node_modules\sequelize\lib\sequelize.js:315:28
    at runNextTicks (node:internal/process/task_queues:60:5)
    at process.processTimers (node:internal/timers:509:9)
    at async PostgresQueryInterface.select (D:\Test\chartbrew\server\node_modules\sequelize\lib\dialects\abstract\query-interface.js:407:12)
    at async Chart.findAll (D:\Test\chartbrew\server\node_modules\sequelize\lib\model.js:1140:21) {
  name: 'SequelizeDatabaseError',
  parent: error: relation "Chart" does not exist
      at Parser.parseErrorMessage (D:\Test\chartbrew\server\node_modules\pg-protocol\dist\parser.js:287:98)
      at Parser.handlePacket (D:\Test\chartbrew\server\node_modules\pg-protocol\dist\parser.js:126:29)
      at Parser.parse (D:\Test\chartbrew\server\node_modules\pg-protocol\dist\parser.js:39:38)
      at Socket.<anonymous> (D:\Test\chartbrew\server\node_modules\pg-protocol\dist\index.js:11:42)
      at Socket.emit (node:events:514:28)
      at addChunk (node:internal/streams/readable:324:12)
      at readableAddChunk (node:internal/streams/readable:297:9)
      at Readable.push (node:internal/streams/readable:234:10)
      at TCP.onStreamRead (node:internal/stream_base_commons:190:23) {
    length: 106,
    severity: 'ERROR',
    code: '42P01',
    detail: undefined,
    hint: undefined,
    position: '1359',
    internalPosition: undefined,
    internalQuery: undefined,
    where: undefined,
    schema: undefined,
    table: undefined,
    column: undefined,
    dataType: undefined,
    constraint: undefined,
    file: 'parse_relation.c',
    line: '1452',
    routine: 'parserOpenTable',
    sql: 'SELECT "Chart"."id", "Chart"."project_id", "Chart"."name", "Chart"."lastAutoUpdate", "Chart"."autoUpdate", "Chart"."chartData", "ChartDatasetConfigs"."id" AS "ChartDatasetConfigs.id", "ChartDatasetConfigs"."chart_id" AS "ChartDatasetConfigs.chart_id", "ChartDatasetConfigs"."dataset_id" AS "ChartDatasetConfigs.dataset_id", "ChartDatasetConfigs"."formula" AS "ChartDatasetConfigs.formula", "ChartDatasetConfigs"."datasetColor" AS "ChartDatasetConfigs.datasetColor", "ChartDatasetConfigs"."fillColor" AS "ChartDatasetConfigs.fillColor", "ChartDatasetConfigs"."fill" AS "ChartDatasetConfigs.fill", "ChartDatasetConfigs"."multiFill" AS "ChartDatasetConfigs.multiFill", "ChartDatasetConfigs"."legend" AS "ChartDatasetConfigs.legend", "ChartDatasetConfigs"."pointRadius" AS "ChartDatasetConfigs.pointRadius", "ChartDatasetConfigs"."excludedFields" AS "ChartDatasetConfigs.excludedFields", "ChartDatasetConfigs"."sort" AS "ChartDatasetConfigs.sort", "ChartDatasetConfigs"."columnsOrder" AS "ChartDatasetConfigs.columnsOrder", "ChartDatasetConfigs"."order" AS "ChartDatasetConfigs.order", "ChartDatasetConfigs"."maxRecords" AS "ChartDatasetConfigs.maxRecords", "ChartDatasetConfigs"."goal" AS "ChartDatasetConfigs.goal", "ChartDatasetConfigs"."createdAt" AS "ChartDatasetConfigs.createdAt", "ChartDatasetConfigs"."updatedAt" AS "ChartDatasetConfigs.updatedAt" FROM "Chart" AS "Chart" LEFT OUTER JOIN "ChartDatasetConfig" AS "ChartDatasetConfigs" ON "Chart"."id" = "ChartDatasetConfigs"."chart_id" WHERE "Chart"."autoUpdate" > 0;',
    parameters: undefined
  },
  original: error: relation "Chart" does not exist
      at Parser.parseErrorMessage (D:\Test\chartbrew\server\node_modules\pg-protocol\dist\parser.js:287:98)
      at Parser.handlePacket (D:\Test\chartbrew\server\node_modules\pg-protocol\dist\parser.js:126:29)
      at Parser.parse (D:\Test\chartbrew\server\node_modules\pg-protocol\dist\parser.js:39:38)
      at Socket.<anonymous> (D:\Test\chartbrew\server\node_modules\pg-protocol\dist\index.js:11:42)
      at Socket.emit (node:events:514:28)
      at addChunk (node:internal/streams/readable:324:12)
      at readableAddChunk (node:internal/streams/readable:297:9)
      at Readable.push (node:internal/streams/readable:234:10)
      at TCP.onStreamRead (node:internal/stream_base_commons:190:23) {
    length: 106,
    severity: 'ERROR',
    code: '42P01',
    detail: undefined,
    hint: undefined,
    position: '1359',
    internalPosition: undefined,
    internalQuery: undefined,
    where: undefined,
    schema: undefined,
    table: undefined,
    column: undefined,
    dataType: undefined,
    constraint: undefined,
    file: 'parse_relation.c',
    line: '1452',
    routine: 'parserOpenTable',
    sql: 'SELECT "Chart"."id", "Chart"."project_id", "Chart"."name", "Chart"."lastAutoUpdate", "Chart"."autoUpdate", "Chart"."chartData", "ChartDatasetConfigs"."id" AS "ChartDatasetConfigs.id", "ChartDatasetConfigs"."chart_id" AS "ChartDatasetConfigs.chart_id", "ChartDatasetConfigs"."dataset_id" AS "ChartDatasetConfigs.dataset_id", "ChartDatasetConfigs"."formula" AS "ChartDatasetConfigs.formula", "ChartDatasetConfigs"."datasetColor" AS "ChartDatasetConfigs.datasetColor", "ChartDatasetConfigs"."fillColor" AS "ChartDatasetConfigs.fillColor", "ChartDatasetConfigs"."fill" AS "ChartDatasetConfigs.fill", "ChartDatasetConfigs"."multiFill" AS "ChartDatasetConfigs.multiFill", "ChartDatasetConfigs"."legend" AS "ChartDatasetConfigs.legend", "ChartDatasetConfigs"."pointRadius" AS "ChartDatasetConfigs.pointRadius", "ChartDatasetConfigs"."excludedFields" AS "ChartDatasetConfigs.excludedFields", "ChartDatasetConfigs"."sort" AS "ChartDatasetConfigs.sort", "ChartDatasetConfigs"."columnsOrder" AS "ChartDatasetConfigs.columnsOrder", "ChartDatasetConfigs"."order" AS "ChartDatasetConfigs.order", "ChartDatasetConfigs"."maxRecords" AS "ChartDatasetConfigs.maxRecords", "ChartDatasetConfigs"."goal" AS "ChartDatasetConfigs.goal", "ChartDatasetConfigs"."createdAt" AS "ChartDatasetConfigs.createdAt", "ChartDatasetConfigs"."updatedAt" AS "ChartDatasetConfigs.updatedAt" FROM "Chart" AS "Chart" LEFT OUTER JOIN "ChartDatasetConfig" AS "ChartDatasetConfigs" ON "Chart"."id" = "ChartDatasetConfigs"."chart_id" WHERE "Chart"."autoUpdate" > 0;',
    parameters: undefined
  },
  sql: 'SELECT "Chart"."id", "Chart"."project_id", "Chart"."name", "Chart"."lastAutoUpdate", "Chart"."autoUpdate", "Chart"."chartData", "ChartDatasetConfigs"."id" AS "ChartDatasetConfigs.id", "ChartDatasetConfigs"."chart_id" AS "ChartDatasetConfigs.chart_id", "ChartDatasetConfigs"."dataset_id" AS "ChartDatasetConfigs.dataset_id", "ChartDatasetConfigs"."formula" AS "ChartDatasetConfigs.formula", "ChartDatasetConfigs"."datasetColor" AS "ChartDatasetConfigs.datasetColor", "ChartDatasetConfigs"."fillColor" AS "ChartDatasetConfigs.fillColor", "ChartDatasetConfigs"."fill" AS "ChartDatasetConfigs.fill", "ChartDatasetConfigs"."multiFill" AS "ChartDatasetConfigs.multiFill", "ChartDatasetConfigs"."legend" AS "ChartDatasetConfigs.legend", "ChartDatasetConfigs"."pointRadius" AS "ChartDatasetConfigs.pointRadius", "ChartDatasetConfigs"."excludedFields" AS "ChartDatasetConfigs.excludedFields", "ChartDatasetConfigs"."sort" AS "ChartDatasetConfigs.sort", "ChartDatasetConfigs"."columnsOrder" AS "ChartDatasetConfigs.columnsOrder", "ChartDatasetConfigs"."order" AS "ChartDatasetConfigs.order", "ChartDatasetConfigs"."maxRecords" AS "ChartDatasetConfigs.maxRecords", "ChartDatasetConfigs"."goal" AS "ChartDatasetConfigs.goal", "ChartDatasetConfigs"."createdAt" AS "ChartDatasetConfigs.createdAt", "ChartDatasetConfigs"."updatedAt" AS "ChartDatasetConfigs.updatedAt" FROM "Chart" AS "Chart" LEFT OUTER JOIN "ChartDatasetConfig" AS "ChartDatasetConfigs" ON "Chart"."id" = "ChartDatasetConfigs"."chart_id" WHERE "Chart"."autoUpdate" > 0;',
  parameters: {}
}

**Comment by razvanilin** [comment]
Sorry I missed this! Can you tell me what version of Chartbrew you're trying to set up?

**Comment by razvanilin** [comment]
There were some issues with the migrations on Windows, but this has now been solved in v3.4.0
